aten/src/ATen/native/mps/operations/Quantized.mm

```diff
@@ -758,6 +763,38 @@ Tensor _convert_weight_to_int4pack_mps(const Tensor& in, int64_t innerKTiles) {
   return weight_packed;
 }
 
+Tensor _convert_scales_and_zeros_mps(const Tensor& scales, const Tensor& zeros) {
+  static const bool is_macOS_15_0_or_newer = is_macos_13_or_newer(MacOSVersion::MACOS_VER_15_0_PLUS);
+  if (is_macOS_15_0_or_newer) {
+    return at::stack({scales, zeros}).contiguous();
+  } else {
+    return at::transpose(at::cat({scales, zeros}, /*dim=*/2), 0, 1).contiguous();
+  }
+}
```

Comment:
This can go in `torch/backends/mps/__init__.py` - I followed the same style as `_convert_weight_to_int4pack` was implemented thus the reason of adding to native_functions.yaml.